app/ts/types/JsonRpc-types.ts

```diff
@@ -357,6 +357,12 @@ export const GetFilterLogs = funtypes.ReadonlyObject({
 	params: funtypes.ReadonlyTuple(funtypes.String)
 })
 
+export type EthChainId = funtypes.Static<typeof EthChainId>
+export const EthChainId = funtypes.ReadonlyObject({
+	method: funtypes.Literal('eth_chainId'),
+	params: funtypes.ReadonlyArray(funtypes.Never),
```

Comment:
params field is not mandatory for `eth_chainId`

Reply:
removed as the inline definition will already suffice (missed that one out)